tests/unit/cluster/announce-client-ip.tcl

```diff
@@ -147,3 +147,15 @@ start_cluster 2 2 {tags {external:skip cluster ipv6} overrides {cluster-replica-
         [lindex $clients $j] close
     }
 }
+
+start_cluster 1 0 {tags {external:skip cluster ipv6} overrides {cluster-replica-no-failover yes bind {127.0.0.1 ::1}}} {
```

Comment:
let 's split it into two start_cluster? One for ipv4 and one for ipv6 (and tag it with ipv6 so we can skip it in some cases, see #910)